cylc/flow/network/resolvers.py

```diff
@@ -711,6 +713,9 @@ async def mutator(
             return [{
                 'response': (False, f'No matching workflow in {workflows}')}]
         w_id = w_ids[0]
+        # For back compat, with gql-v3 None will not use default_value.
```

Comment:
We use `BACK COMPAT` comments to make code like this so we can easily grep for it later:

```suggestion
        # BACK COMPAT: transform "None" to "[]"
        # url: https://github.com/cylc/cylc-flow/pull/6478
        # from: <8.5.0
        # to: >=8.5.0
        # remove at: 8.x
         For back compat, with gql-v3 None will not use default_value.
```